<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>statsviewer</class>
 <widget class="QWidget" name="statsviewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>180</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>OpenBKZ Statistics</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="8">
    <widget class="QComboBox" name="titleBox">
     <property name="maximumSize">
      <size>
       <width>95</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Title</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="8">
    <widget class="QComboBox" name="statsTypeBox">
     <property name="maximumSize">
      <size>
       <width>95</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Overview</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Journal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Times Per Page</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Rating(s)</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="8">
    <widget class="QPushButton" name="timesToggle">
     <property name="maximumSize">
      <size>
       <width>95</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>View Dates</string>
     </property>
    </widget>
   </item>
   <item row="6" column="8">
    <widget class="QSlider" name="zoomSlider">
     <property name="sliderPosition">
      <number>80</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="titleDateLabel">
     <property name="minimumSize">
      <size>
       <width>550</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>550</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string>Title | Start Date to End Date</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>17</height>
      </size>
     </property>
     <property name="text">
      <string>OpenBKZ Statisics Alpha v0.4</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" rowspan="5" colspan="8">
    <widget class="QGraphicsView" name="graphicsView"/>
   </item>
   <item row="10" column="1">
    <widget class="QLabel" name="avg_1">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>180</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Avg time per page: </string>
     </property>
    </widget>
   </item>
   <item row="10" column="2" colspan="4">
    <widget class="QSlider" name="stretchSlider">
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>0</height>
      </size>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
     <property name="pageStep">
      <number>2</number>
     </property>
     <property name="sliderPosition">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
